README.adoc

```diff
@@ -392,9 +392,9 @@ Define the [hotspot=systemContainer file=2]`systemContainer` test container to s
 
 The [hotspot=testcontainers file=2]`@Testcontainers` annotation finds all fields that are annotated with the [hotspot=container file=2]`@Container` annotation and calls their container lifecycle methods. The `static` function declaration on each container indicates that this container will be started only once before any test method is executed and stopped after the last test method is executed.
 
-The [hotspot=testGetSystem file=2]`testGetSystem()` verifies the `/query/system/{hostname}` endpoint with `hostname` specified to `system-java8`.
+The [hotspot=testGetSystem file=2]`testGetSystem()` verifies the `/query/system/{hostname}` endpoint with `hostname` specified as `system-java8`.
```

Comment:
See prev comment about specified vs set and the wording of the lead-in to the example.